### DISCUSSION TITLE
Welcome to the ENJINN help forum

### DISCUSSION THREAD

**Discussion by shayded-exe** [rshea0/enjinn#2] [General]
Hey y'all. This is a place to ask any questions you have about how to use ENJINN or post any issues you have.

I check this frequently and will respond quickly to anything that needs attention so don't hesitate to ask.

**This thread is for general comments. For ideas/issues please open a [new thread](https://github.com/rshea0/enjinn/discussions/new)**

**Comment by JONESYdj** [comment]
Hi rshea0,
I'm having an issue making your tool to work. I have already been able to locate the Engine Library and have created the enjinn.yaml file but the tool doesn't find it.
<img width="750" alt="Screenshot 2021-05-27 at 17 08 55" src="https://user-images.githubusercontent.com/84925705/119860432-397ef100-bf0e-11eb-9879-8bd8c0cced35.png">
<img width="763" alt="Screenshot 2021-05-27 at 17 09 55" src="https://user-images.githubusercontent.com/84925705/119860569-5d423700-bf0e-11eb-89df-ad96523b20ce.png">

I have created the file manually with Text Edit (unicode).
What am I doing wrong?

**Reply by shayded-exe** [reply]
Ah, I think Unicode might be the problem if it's UTF-16. Could you please email it to shayded@shayded.com?

Try converting the file to UTF-8 and try again. In the GUI version, I'll have it create the config file for you so this isn't an issue.
